diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+GitHub is an online repository hosting service that stores Git repositories, which are distributed version control software.
```

Comment:
Be careful not to mix up Git & Github. Git repositories are not distributed version control software.
Github is a user interface (UI) for accessing git's version control capabilities for our project repositories.